plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -560,4 +561,22 @@ public InputStream getPropertiesInputStream(String path) {
     public boolean isShowVersion() {
         return showVersion;
     }
+
+
+    /**
+     * Determines the hash function used with the Update Center
+     * set via environment variable only
+     *
+     * @return the string value for the hash function. Currently allows sha1, sha256(default), sha512
+     */
+    private String getHashFunction() {
```

Comment:
Not a proper review, but would try using an `enum`, would make also the if/else easier.

Reply:
I'll update that, sure. I realize that actually helps with the command parsing and fails quickly.

Reply:
See latest PR.